### DISCUSSION TITLE
Archived versions of SDK

### DISCUSSION THREAD

**Discussion by PippoApps** [airsdk/Adobe-Runtime-Support#1035] [Q&A]
Hello there,
is there a way to download old versions of the sdk?
I need to get hold of 33.1.1.444
cheers

**Comment by leossmith** [comment]
Go to the latest link to download and replace the 33.1.1.554 with the version you want.

**Reply by ajwfrost** [reply]
Yes or the available ones are presented as hyperlinks on https://airsdk.harman.com/release_notes

So: https://airsdk.harman.com/download/33.1.1.444

**Comment by PippoApps** [comment]
Thank you @ajwfrost .
Silly me I clicked on the header and opened the release notes, while clicking on the version number goes to the relevant page.
Cheers

**Comment by marchbold** [comment]
You can use APM now too.

```
apm airsdk install 33.1.1.444
```

This will download and extract the sdk to the current directory.

More info https://github.com/airsdk/apm/wiki/Usage-AIRSDK

**Reply by PippoApps** [reply]
Thank you!
